api/models/internal.go

```diff
@@ -0,0 +1,52 @@
+package models
+
+import (
+	"fmt"
+	"github.com/raintank/metrictank/cluster"
+	"github.com/raintank/metrictank/consolidation"
+	"github.com/raintank/metrictank/util"
+)
+
+type Req struct {
+	// these fields can be set straight away:
+	Key          string                     `json:"key"`    // metric key aka metric definition id (orgid.<hash>), often same as target for graphite-metrictank requests
+	Target       string                     `json:"target"` // the target we should return either to graphite or as if we're graphite.  this is usually the original input string like consolidateBy(key,'sum') except when it's a pattern than it becomes the concrete value like consolidateBy(foo.b*,'sum') -> consolidateBy(foo.bar,'sum') etc
+	Node         *cluster.Node              `json:"-"`      // which instance to request from. tcp addr or 'local'
```

Comment:
it seems like cluster.Node is a type that aims to hold all information for a node to describe itself, including statechange timestamps, version and so on.  It doesn't seem like a good fit for the purpose of merely describing which node a a req should be executed on.
also, since this is part of the clustering changeset, why don't we remove this property out of this PR and then introduce it (or something more appropriate) in the cluster PR